.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb-base",
+  "env": {
+    "es6": true,
+    "browser": true,
+    "jasmine": true,
```

Comment:
We would prefer to use Intern. Intern 4 which will be released this year has been refactored using TS which will make testing ES code without transpilation easy enough to achieve.